libs/designer/src/lib/core/state/staticresultschema/staticresultsSlice.ts

```diff
@@ -25,23 +25,36 @@ export const staticResultsSlice = createSlice({
     initializeStaticResultProperties: (state, action: PayloadAction<Record<string, any>>) => {
       state.properties = action.payload;
     },
+    initScopeCopiedStaticResultProperties: (state, action: PayloadAction<Record<string, any>>) => {
+      const copiedConnections = action.payload;
+      Object.entries(copiedConnections).forEach(([nodeId, staticResultProperty]) => {
+        state.properties[nodeId + 0] = staticResultProperty;
```

Comment:
I'm confused on the `nodeId + 0` here

Reply:
This was just how old designer stored static results, I'm not sure why. But all it was was always just stored as nodeId + 0. So if a node was like parseJson, its staticresult property is stored in parseJson0

Reply:
Not too big of a fan of this, but I think we need to keep it like so to maintain backwards compatibility